src/components/SimpleList/components/SimpleItem.js

```diff
@@ -40,6 +43,7 @@ const SimpleListItem = ({
   flex,
   align,
   column,
+  rowIndex,
```

Comment:
rowIndex help uniqueify the simpleItems in terms of the row they are children of, this is useful for when cypress needs to target a simpleitem in a specific row

Reply:
refactored this out infavour of a customiseable Row component